meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -31,8 +32,13 @@ do_install_append() {
 
     install -d ${D}${systemd_unitdir}/system/
     install -m 0644 ${WORKDIR}/ambd.service ${D}${systemd_unitdir}/system
+
+    install -d ${D}${libdir}/systemd/user/
+    install -m 0644 ${WORKDIR}/ambd_fsa.service ${D}${libdir}/systemd/user
```

Comment:
Why does this service need to run as part of user instance while AMBD itself runs on system instance?